code/datums/ert.dm

```diff
@@ -53,3 +53,12 @@
 	rename_team = "Inquisition"
 	mission = "Destroy any traces of paranormal activity aboard the station."
 	polldesc = "a Nanotrasen paranormal response team"
+
+/datum/ert/janitor
+	roles = list(/datum/antagonist/ert/janitor, /datum/antagonist/ert/janitor/heavy)
+	leader_role = /datum/outfit/ert/janitor/heavy
+	teamsize = 4
+	opendoors = FALSE
+	rename_team = "Janitor"
+	mission = "Clean up EVERYTHING."
+	polldesc = "a Nanotrasen janitorial response team"
```

Comment:
Capitalize response

Reply:
for what reason though? it's not capitalized in other places

Reply:
```suggestion
	polldesc = "a Nanotrasen Janitorial Response Team"
```
https://github.com/tgstation/tgstation/pull/42659/files#diff-d99f27400b888b7587642f5bdfea3770R36
https://github.com/tgstation/tgstation/pull/42659/files#diff-d99f27400b888b7587642f5bdfea3770R45

Reply:
guess i'm blind